docs/proposals/images/mhc.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could this be regenerated using plantuml like the other diagrams? That way it would be easier to modify/update as needed in the future.